### DISCUSSION TITLE
cn/2024/09/02/limit-ones-freedom/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#179] [Announcements]
# cn/2024/09/02/limit-ones-freedom/

https://hongtaoh.com/cn/2024/09/02/limit-ones-freedom/

**Comment by XiangyunHuang** [comment]
写得真好，一起共勉。

**Reply by XiangyunHuang** [reply]
我感觉得经常写反思，因为人性都是朝着腐化堕落的方向滑。

**Reply by hongtaoh** [reply]
对啊。《[少有人走的路](https://hongtaoh.com/cn/2022/09/04/road-less-traveled/)》里面开篇讲的就是自律。人性是往下沉的，没有自律没办法往上走。
